### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by kikollovet**
Everything done according to the issue. Used ember simple auth addon. 

Just 1 other issue appeard. Its due to ember simple auth. Whenever you pick a route that is protected you are redirected to the login form. If you reload the login form ember simple auth will **'forget'** the last attempted route and doing so if you login you are going to be redirected to the route '/'. This issue only happens if you do the reload when in the login form. You can type wrong the password as many times as you want and still ember simple auth will **'remeber'** the last attempted route and redirect you to it when you login.

### CODE REVIEW

File: .firebase/hosting.ZGlzdA.cache
Comment (tpetrone) [inline .firebase/hosting.ZGlzdA.cache]: This file should be ignored by your version control system

File: app/controllers/login.js
Comment (tpetrone) [inline app/controllers/login.js]: remove commented lines. They are of no use in `master` branch

File: app/routes/admin/contacts.js
Comment (tpetrone) [inline app/routes/admin/contacts.js]: remove commented lines

File: app/routes/admin/contacts.js
Comment (tpetrone) [inline app/routes/admin/contacts.js:2]: excellent

File: app/routes/admin/contacts.js
Comment (tpetrone) [inline app/routes/admin/contacts.js:7]: better than before ;)

File: app/routes/application.js
Comment (tpetrone) [inline app/routes/application.js:11]: several idention problems. **always** indent your code properly

File: app/routes/loginsemuso.js
Comment (tpetrone) [inline app/routes/loginsemuso.js]: which file is this???

File: app/templates/login.hbs
Comment (tpetrone) [inline app/templates/login.hbs:3]: indentation problem

File: config/environment.js
Comment (tpetrone) [inline config/environment.js]: commented lines

File: tests/unit/routes/login-test.js
Comment (tpetrone) [inline tests/unit/routes/login-test.js]: remove file

**Review by tpetrone — CHANGES_REQUESTED** [review]
Pls provide sign in information

File: app/routes/loginsemuso.js
Comment (kikollovet) [reply app/routes/loginsemuso.js]: I was using a route instead of a controller. I changed to the controller so I could use the error message. Just deleted unused route.

**Comment by kikollovet** [comment]
Petrone, I made a service because I tought you want the app to show sign in info of the user. It´s easy to delete the service. I will leave this way for now. The only problem is the navbar looks uglier with the sign in info because it gets bigger...... Now I tought better and I think you wanted this: user@teste.com pass: testeteste ------- library-app-d127b.web.app/.

File: .gitignore
Comment (tpetrone) [inline .gitignore:21]: This won't be enough.
Take a look at
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

File: app/controllers/login.js
Comment (tpetrone) [inline app/controllers/login.js]: still indentation problems in this file. This is **super** important. review all your file's identation

File: tests/unit/services/logged-user-test.js
Comment (tpetrone) [inline tests/unit/services/logged-user-test.js]: remove unused test

**Review by tpetrone — COMMENTED** [review]
I didn't understand what you meant by
```
Petrone, I made a service because I tought you want the app to show sign in info of the user. It´s easy to delete the service. I will leave this way for now. The only problem is the navbar looks uglier with the sign in info because it gets bigger...... 
```

Service was the way to go indeed

**Comment by kikollovet** [comment]
So I didn´t misunderstood you. You asked to show sign in info when the user was logged in. I thought that maybe when
 you said 'give me sign in info', you  meant to give you a user and a password so you could log in...... but I was wrong.

File: .gitignore
Comment (kikollovet) [reply .gitignore:21]: I got it, I will remove it with git rm --cached file after we finish with this merge pull request!

File: app/controllers/login.js
Comment (kikollovet) [reply app/controllers/login.js]: All files reviwed!

**Comment by tpetrone** [comment]
> So I didn´t misunderstood you. You asked to show sign in info when the user was logged in. I thought that maybe when
> you said 'give me sign in info', you meant to give you a user and a password so you could log in...... but I was wrong.

@kikollovet That is exactly what I meant. I needed the login info so I could navigate over the app. I created an issue #8 to use README.md to document those infos

File: .gitignore
Comment (tpetrone) [reply .gitignore:21]: @kikollovet You should remove now, commit, and push. Waiting for this so I can merge

**Comment by kikollovet** [comment]
So, do I keep the service and the email showing of the user logged in the app?

**Comment by kikollovet** [comment]
> 
> 
> > So I didn´t misunderstood you. You asked to show sign in info when the user was logged in. I thought that maybe when
> > you said 'give me sign in info', you meant to give you a user and a password so you could log in...... but I was wrong.
> 
> @kikollovet That is exactly what I meant. I needed the login info so I could navigate over the app. I created an issue #8 to use README.md to document those infos

So, do I keep the service and the email showing of the user logged in the app?

**Comment by kikollovet** [comment]
So, after you merge, I have to do git pull origin master in my git bash and my local repository will  update and be the same as the remote repository?

**Comment by kikollovet** [comment]
> 
> 
> > So I didn´t misunderstood you. You asked to show sign in info when the user was logged in. I thought that maybe when
> > you said 'give me sign in info', you meant to give you a user and a password so you could log in...... but I was wrong.
> 
> @kikollovet That is exactly what I meant. I needed the login info so I could navigate over the app. I created an issue #8 to use README.md to document those infos

@tpetrone I will solve this issue next since its very quick

**Pull request merged**
